Kitodo/src/main/java/org/kitodo/production/helper/batch/BatchTaskHelper.java

```diff
@@ -455,7 +455,6 @@ private boolean isTaskValid(Task task) throws IOException {
     private boolean invalidMetadataExists(Task task) {
         if (task.isTypeMetadata()
                 && ConfigCore.getBooleanParameterOrDefaultValue(ParameterCore.USE_META_DATA_VALIDATION)) {
-            ServiceManager.getMetadataValidationService().setAutoSave(true);
```

Comment:
why do you remove it? Has it something to do with validation?

Reply:
Here, a variable is set, which has no further meaning. It is not used anywhere.

Reply:
I'm just wondering, why you changed it in this PR